app/pb_kits/playbook/pb_selectable_card/_selectable_card.html.erb

```diff
@@ -0,0 +1,18 @@
+<%= content_tag(:div,
+  data: object.data,
+  class: object.classname) do %>
+
+  <% if object.multi %>
+    <%= check_box_tag object.name, object.value, object.checked, id: object.html_for, disabled: object.disabled %>
+  <% else %>
+    <%= radio_button_tag object.name, object.value, object.checked, id: object.html_for, disabled: object.disabled %>
+  <% end %>
```

Comment:
@markiearnold & @thestephenmarshall I do not believe these cards will be drop-in unless we can somehow let the user pass their own radio or checkbox. Perhaps we assume it's part of the children if a certain prop is passed?

Reply:
That is how I would assume @jasoncypret